dev/README.md

```diff
@@ -125,17 +164,29 @@ To deposit money into your account:
 
 ## Troubleshooting
 
-### Distribution account out of funds
+#### Sample Tenant Management Postman collection
```

Comment:
This section isn't needed anymore if we're removing the Postman collection from this PR. But it would be good to keep it in a separate branch so that it's not lost.